src/pages/data-reporting/reports-api/index.mdx

```diff
@@ -520,7 +520,9 @@ https://api.livechatinc.com/v3.4/reports/chats/engagement \
 
 ### Greetings Conversion
 
-Shows the number of greetings sent to the customers and how many of those resulted in a chat or a goal. Greetings are also known as targeted messages.
+Shows the number of greetings sent to the customers and how many of those resulted in a chat or a goal. Greetings are also known as targeted messages. 
```

Comment:
@ale-kacperczyk this PR is connected with the [issue](https://livechat.slack.com/archives/C2XMTMQ5S/p1661350932696619) reported on #platform-docs. I discussed it with Jay and Wojtek Zbrożek who raised it 3 months ago. Unfortunately, Jan Owsiany who was explaining the solution to Wojtek is no longer in LiveChat. However, Wojtek confirmed that, in terms of content, with this part added clients should not have any doubts.

Please, let me know if I should also confirm this with someone else :)